docs/dashboards/datagrid-component.md

```diff
@@ -0,0 +1,104 @@
+DataGrid Component
+===
+
+The DataGrid Component can be placed inside a data cell to allow the data visualization in the form of an editable data grid.
+
+<iframe style="width: 100%; height: 470px; border: none;" src="https://www.highcharts.com/samples/nonav/dashboards/datagrid-component/datagrid-options" allow="fullscreen"></iframe>
+
+
+## How to start
+
+1. To use the DataGrid Component in Dashboards, you need to import the appropriate modules.
+    ```html
+    <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+    <script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+    <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+    ```
+
+    Alternatively, you can also use NPM packages (see: [Installation](https://www.highcharts.com/docs/dashboards/installation)) and import to connect them to the Dashboards.
+    ```ts
+    import * as Dashboards from '@highcharts/dashboards';
+    import DataGrid from '@highcharts/dashboards/es-modules/DataGrid/DataGrid';
+    import DataGridPlugin from '@highcharts/dashboards/es-modules/Dashboards/Plugins/DataGridPlugin';
+
+    DataGridPlugin.custom.connectDataGrid(DataGrid);
+    Dashboards.PluginHandler.addPlugin(DataGridPlugin);
+    ```
+
+2. You also need to import the styles.
```

Comment:
Wondering if we should specify "in your main CSS file" here? 🤔

Reply:
Also applies to the HTML and JS I guess, but methinks it's easier to confuse the CSS import for JS